[2025-10-11T14:43:02.845+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-11T14:43:02.856+0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cafe_sales_etl.load_to_postgres manual__2025-10-11T11:42:44.112144+00:00 [queued]>
[2025-10-11T14:43:02.862+0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cafe_sales_etl.load_to_postgres manual__2025-10-11T11:42:44.112144+00:00 [queued]>
[2025-10-11T14:43:02.862+0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-10-11T14:43:02.881+0300] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-10-11 11:42:44.112144+00:00
[2025-10-11T14:43:02.887+0300] {standard_task_runner.py:72} INFO - Started process 61264 to run task
[2025-10-11T14:43:02.890+0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cafe_sales_etl', 'load_to_postgres', 'manual__2025-10-11T11:42:44.112144+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/cafe_sales_etl_dag.py', '--cfg-path', '/tmp/tmp6x2l5srs']
[2025-10-11T14:43:02.891+0300] {standard_task_runner.py:105} INFO - Job 75: Subtask load_to_postgres
[2025-10-11T14:43:02.937+0300] {task_command.py:467} INFO - Running <TaskInstance: cafe_sales_etl.load_to_postgres manual__2025-10-11T11:42:44.112144+00:00 [running]> on host archlinux
[2025-10-11T14:43:03.010+0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cafe_sales_etl' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-10-11T11:42:44.112144+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-11T11:42:44.112144+00:00'
[2025-10-11T14:43:03.012+0300] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-10-11T14:43:03.012+0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-10-11T14:43:03.012+0300] {logging_mixin.py:190} INFO - Current task name:load_to_postgres state:running start_date:2025-10-11 11:43:02.857134+00:00
[2025-10-11T14:43:03.012+0300] {logging_mixin.py:190} INFO - Dag name:cafe_sales_etl and current dag run status:running
[2025-10-11T14:43:03.012+0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-11T14:43:04.121+0300] {logging_mixin.py:190} INFO - âœ… Loaded 7773 rows into PostgreSQL table: sales_data
[2025-10-11T14:43:04.124+0300] {python.py:240} INFO - Done. Returned value was: None
[2025-10-11T14:43:04.131+0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-11T14:43:04.132+0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cafe_sales_etl, task_id=load_to_postgres, run_id=manual__2025-10-11T11:42:44.112144+00:00, execution_date=20251011T114244, start_date=20251011T114302, end_date=20251011T114304
[2025-10-11T14:43:04.146+0300] {logging_mixin.py:190} INFO - Task instance in success state
[2025-10-11T14:43:04.147+0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-10-11T14:43:04.147+0300] {logging_mixin.py:190} INFO - Dag name:cafe_sales_etl queued_at:2025-10-11 11:42:44.126708+00:00
[2025-10-11T14:43:04.147+0300] {logging_mixin.py:190} INFO - Task hostname:archlinux operator:PythonOperator
[2025-10-11T14:43:04.186+0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-11T14:43:04.209+0300] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-11T14:43:04.223+0300] {local_task_job_runner.py:245} INFO - ::endgroup::
